tests/headingcommand.js

```diff
@@ -176,13 +176,13 @@ describe( 'HeadingCommand', () => {
 
 		it( 'should do nothing on non-registered model elements', () => {
```

Comment:
And this test is unnecessary now since it checks integration with the schema (which I guess must be covered somewhere else too).